packages/react-dom/src/events/SyntheticMouseEvent.js

```diff
@@ -53,7 +53,8 @@ const SyntheticMouseEvent = SyntheticUIEvent.extend({
       return 0;
     }
 
-    return event.screenX - screenX;
+    // movementX/Y must be zero for all mouse events except mousemove.
```

Comment:
You can remove this comment (and a similar one below), it's already implied by the code.